jenkins/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# -- DISCLAIMER --
+# This Dockerfile is used by Jenkins to create a testing environment.  It will
+# NOT create a containerized instance of pycds.
+FROM ubuntu:18.04
+
+# Set version for the build stage
+ARG PYTHON_MAJOR=3.6
+ARG PYTHON_PATCH=10
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set up locales
+RUN apt-get update && \
+	apt-get install -y locales && \
+	sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+	dpkg-reconfigure --frontend=noninteractive locales && \
+	update-locale LANG=en_US.UTF-8
+
+ENV LANG en_US.UTF-8
+
+# Set up a repo
```

Comment:
Just out of curiosity, what exactly does this do? I'm not enough of a package expert to understand it off the cuff.

Reply:
So the earlier `postgres-9x` packages aren't available anymore, and you have to go through [PPAs (Personal Package Archive)](https://itsfoss.com/ppa-guide/) to get them.  PPAs are just repos for packages, in this case we are using the repo that still has older version of `postgres`. Setting up the repo allows you to update the `source.list` for `apt` and include packages available in the repo.  So in this case we are getting `postgres-9.4-postgis-2.4` in the list from the ppa before running this install.  `apt` would not recognize the package otherwise.

Reply:
Thanks! :100: